src/bic/view/form/pending.js

```diff
@@ -0,0 +1,173 @@
+define(function (require) {
+  'use strict';
+
+  // foreign modules
+
+  var PendingQueue;
+  var $ = require('jquery');
+  var _ = require('underscore');
+  var Mustache = require('mustache');
+  var Backbone = require('backbone');
+
+  // local modules
+
+  // ...
+
+  // templates
+  var Template = require('text!bic/template/form/pending.mustache');
+  var clearConfirmationTemplate = require('text!bic/template/form/pending-clear-confirmation-popup.mustache');
+
+  // utils
+  var uiTools = require('bic/lib/ui-tools');
+  var MODEL_STATUS = require('bic/enum-model-status');
+
+  // this module
+
+  PendingQueue = Backbone.View.extend({
+
+    events: {
+      'click .clearPendingItem': 'clearPendingItem',
+      'click #submitPendingItems': 'submitPendingItems',
+      'click #clearPendingItems': 'clearPendingItems',
+      'click #clearPendingItemsConfirmation': 'clearPendingItemsConfirmation'
+    },
```

Comment:
we changed the structure of event selector name to `[data-onevent="functionbToCall"]` eg `[data-onclick="clearPendingItem"]`, might as well update this while refactoring :)